Source/WebKit/WebKitSwift/MarketplaceKit/MarketplaceKitWrapper.swift

```diff
@@ -36,10 +36,13 @@ public final class MarketplaceKitWrapper : NSObject {
     @objc
     @available(iOS 17.4, *)
     public static func requestAppInstallation(topOrigin: URL, url: URL) {
-        let metadata = LinkMetadata(referrer: topOrigin, url: url)
         Task { @MainActor in
             do {
-                try await AppLibrary.current.requestAppInstallation(with: metadata)
+#if canImport(MarketplaceKit, _version: "1.4.77.2")
```

Comment:
This seems to be the new API, but we only look for one specific framework version. This seems very likely to break the next time MarketplaceKit updates. Can we not do an "on or after" version check?

I'd be more comfortable with the 'canImport' referring to the version from the first Beta, and assume the user is running newer software (since that will be the right choice for the upcoming Betas and beyond).

Reply:
I spoke with Ben separately, who corrected my understanding that `canImport` does an "on or after" check, not an equality check. so this code is correct.